content/news/playtest-20150408.md

```diff
@@ -0,0 +1,17 @@
+We have finalized the balancing for the Red Alert mod, so it's time for another playtest!
+
+The changes this time around include:
+
+  - More balancing adjustments in Red Alert. Please see [the changelog](http://changelog.openra.net) for a detailed list of changes.
+  - Fixed enemies erroneously getting an "(Ally)" suffix in chat messages.
+  - Password protected games will be indicated as such in the server browser even when the game has already started.
+  - Fixed helicopters trying to shoot at targets directly below them, thus wasting ammo.
```

Comment:
"Fixed helicopters wasting ammo by trying to shoot at targets directly below them." reads a little better IMO.